lib/auth/webauthncli/prompt.go

```diff
@@ -34,9 +34,12 @@ type DefaultPrompt struct {
 	FirstTouchMessage, SecondTouchMessage string
 	PromptCredentialMessage               string
 
-	ctx   context.Context
-	out   io.Writer
-	count int
+	ctx context.Context
+	out io.Writer
+
+	// Count is the number of times a user was
+	// prompted with a touch message.
+	Count int
```

Comment:
Revert this block? pwdlessLoginPrompt doesn't really seem to need DefaultPrompt and I see no other reason to expose the counts.